spaceship/spec/provisioning_profile_spec.rb

```diff
@@ -54,6 +54,30 @@
     end
   end
 
+  describe '#all via xcode api' do
+    around(:all) do |example|
+      switch = ENV['SPACESHIP_AVOID_XCODE_API']
+      example.run
```

Comment:
Ah, good catch. The docs indicate `around` handles this gracefully:
https://www.relishapp.com/rspec/rspec-core/docs/hooks/around-hooks#an-around-hook-continues-to-run-even-if-the-example-throws-an-exception